resources/test/tests/unit/late-test.html

```diff
@@ -10,11 +10,13 @@
 <p>This test simulates an automated test running scenario, where the test
 results emitted by testharness.js may be interpreted after some delay. It is
 intended to demonstrate that in such cases, any additional tests which are
-executed during that delay are <em>not</em> included in the dataset.</p>
+executed during that delay are included in the dataset.</p>
 
-<p>Although these "late" tests are likely an indication of a mistake in test
-design, they cannot be detected deterministically, so in the interest of
-stability, they should be silently tolerated.</p>
+<p>Although these "late tests" are likely an indication of a mistake in test
+design, they are also recorded. Previously, "late tests" were ignored.
+This test changed to assert "late tests" were no longer ignored after
+https://chromium-review.googlesource.com/c/chromium/src/+/4074305 was
```

Comment:
I think this should be a link to this PR

Reply:
That CL links to https://github.com/web-platform-tests/wpt/pull/37299, but it should be this PR, https://github.com/web-platform-tests/wpt/pull/38806.